src/Core/Scheduler/Providers/WMS_Provider.js

```diff
@@ -37,17 +37,21 @@ WMS_Provider.prototype.preprocessDataLayer = function preprocessDataLayer(layer)
     if (!layer.name) {
         throw new Error('layerName is required.');
     }
-    if (!layer.bbox) {
-        throw new Error('bbox is required');
+    if (!layer.extent) {
+        throw new Error('extent is required');
     }
     if (!layer.projection) {
         throw new Error('projection is required');
     }
 
     layer.extent = new Extent(
         layer.projection,
-        layer.bbox[0], layer.bbox[1],
-        layer.bbox[2], layer.bbox[3]);
+        layer.extent.west, layer.extent.east,
```

Comment:
I think you can directly write: `layer.extent = new Extent(layer.projection, layer.extent);`